modules/cnv-upgrading-cnv.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * cnv/upgrading-container-native-virtualization.adoc
+
+[id="cnv-upgrading-cnv_{context}"]
+= Upgrading {CNVProductName} to the next minor version
+
+You can manually upgrade {CNVProductName} to the next minor version by using the
+{product-title} web console to change the channel of your Operator subscription.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Access the {product-title} web console and navigate to *Operators* -> *Installed Operators*.
+
+. Click *{CNVProductNameStart}* to open the *Operator Details* page.
+
+. Click the *Subscription* tab to open the *Subscription Overview* page.
+
+. Locate the *Channel* pane. Click the pencil icon on the right side of the
```

Comment:
"In the *Channel* pane, click the pencil.."
(Total nit but it makes it sound like I've have to go hunting to find it)